sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyClientBuilder.java

```diff
@@ -26,27 +26,34 @@
 import java.util.Objects;
 
 /**
- * This class provides a fluent builder API to help aid the configuration and instantiation of the {@link CryptographyAsyncClient cryptography async client} and {@link CryptographyClient cryptography sync client},
- * by calling {@link CryptographyClientBuilder#buildAsyncClient() buildAsyncClient} and {@link CryptographyClientBuilder#buildClient() buildClient} respectively
+ * This class provides a fluent builder API to help aid the configuration and instantiation of the {@link
+ * CryptographyAsyncClient cryptography async client} and {@link CryptographyClient cryptography sync client},
+ * by calling {@link CryptographyClientBuilder#buildAsyncClient() buildAsyncClient} and {@link
+ * CryptographyClientBuilder#buildClient() buildClient} respectively
  * It constructs an instance of the desired client.
  *
- * <p> The minimal configuration options required by {@link CryptographyClientBuilder cryptographyClientBuilder} to build {@link CryptographyAsyncClient}
- * are ({@link JsonWebKey jsonWebKey} or {@link String jsonWebKey identifier}) and {@link TokenCredential credential}). </p>
+ * <p> The minimal configuration options required by {@link CryptographyClientBuilder cryptographyClientBuilder} to
+ * build {@link CryptographyAsyncClient}
+ * are ({@link JsonWebKey jsonWebKey} or {@link String jsonWebKey identifier}) and {@link TokenCredential credential}).
+ * </p>
  *
  * {@codesnippet com.azure.security.keyvault.keys.cryptography.async.cryptographyclient.instantiation}
  *
  * <p>The {@link HttpLogDetailLevel log detail level}, multiple custom {@link HttpLoggingPolicy policies} and custom
  * {@link HttpClient http client} can be optionally configured in the {@link CryptographyClientBuilder}.</p>
-
+ 
```

Comment:
Is this intentional? Seems like a line with only a space in it in the middle of the javadoc